docs/usage/configuration-options.md

```diff
@@ -663,7 +663,7 @@ As this is a template it can be dynamically set. E.g. add the `packageName` as p
 ### format
 
 Defines which format the API is returning.
-Only `json` is supported, but more are planned for future.
+At this moment, `json` & `plain` are supported, but more formats are planned for the future
```

Comment:
```suggestion
The API returns the `json` or `plain` format types.
```

Changes:

- Replace `&` with plain English `and`
- Remove "promise" to support more format types in future

I think it's best to say what we support _now_, and leave any future things be.

Question: how do you set the `format` you get from the API? I don't see a code example here. Or is this something that's obvious to "real programmers"? 😄

Reply:
@HonkingGoose I think you can get the examples from here https://github.com/renovatebot/renovate/pull/23563/files#diff-be0a39f342380493f17c380bbd694f1f05d18bdd8d7cecf174369084952102deR646